frontend/public/components/edit-yaml.jsx

```diff
@@ -183,6 +183,7 @@ export const EditYAML_ = connect(stateToProps)(
               !_.isEqual(this.state.fileUpload, nextProps.fileUpload),
               nextProps.fileUpload,
             );
+            this.setState({ fileUpload: nextProps.fileUpload });
```

Comment:
Is there a reason we need to set this in both state and props or can we just read it from props?

Reply:
@spadgett in the constructor we set state = to fileUpload, so I wanted to make sure that we keep state up to date with the latest data if used elsewhere, but it looks like its not referenced anywhere else. So I'm going to remove the state var altogether.